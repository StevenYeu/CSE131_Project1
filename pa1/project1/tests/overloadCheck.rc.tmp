Error, "project1/tests/overloadCheck.rc": 
  Duplicate declaration of overloaded function foo.
Error, "project1/tests/overloadCheck.rc": 
  Duplicate declaration of overloaded function foo.
Compile: failure.
