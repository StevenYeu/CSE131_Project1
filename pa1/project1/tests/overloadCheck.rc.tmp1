  Duplicate declaration of overloaded function foo.
  Duplicate declaration of overloaded function foo.
Compile: failure.
