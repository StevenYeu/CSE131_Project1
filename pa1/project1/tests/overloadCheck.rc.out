Error, "/tmp/cs131_testrunner_QQ0qMF/file.rc": 
  Duplicate declaration of overloaded function foo.
Error, "/tmp/cs131_testrunner_QQ0qMF/file.rc": 
  Duplicate declaration of overloaded function foo.
Compile: failure.
